src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -313,26 +314,30 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
                     })
                   }
                 >
-                  <ul className="govuk-list">
-                    {[
-                      ...otherPublicationReleases.map(({ id, slug, title }) => (
-                        <li key={id} data-testid="other-release-item">
-                          <Link
-                            to={`/find-statistics/${release.publication.slug}/${slug}`}
-                          >
-                            {title}
-                          </Link>
-                        </li>
-                      )),
-                      ...release.publication.legacyReleases.map(
-                        ({ id, description, url }) => (
-                          <li key={id} data-testid="other-release-item">
-                            <a href={url}>{description}</a>
-                          </li>
+                  <ScrollableContainer maxHeight={300}>
+                    <ul className="govuk-list" style={{ overflowY: 'scroll' }}>
```

Comment:
The overflow should be applied in the `ScrollableContainer` component (see other comment)